ModernWpf/Styles/Button.xaml

```diff
@@ -27,26 +28,49 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="Button">
-                    <Border
+                    <Grid>
```

Comment:
Important Note : The PlaneratorHelper places the Planerator within a **Panel** control (It's designed that way sorry!) so be careful when using this effect